llms/mlx_lm/tuner/trainer.py

```diff
@@ -143,9 +143,20 @@ def train(
 ):
     print(f"Starting training..., iters: {args.iters}")
 
+    def check_checkpoints_path(adapter_file):
+        checkpoints_path = "checkpoints"
+        if os.path.dirname(adapter_file):
+            checkpoints_path = os.path.join(
+                os.path.dirname(adapter_file), "checkpoints"
+            )
+
+        if not os.path.exists(checkpoints_path):
+            os.makedirs(checkpoints_path)
+
+        return checkpoints_path
```

Comment:
Do you mind switching this to `pathlib.Path` instead of `os`? We use that everywhere else for the most part and I think it would be good to be consistent.

Reply:
> Do you mind switching this to `pathlib.Path` instead of `os`? We use that everywhere else for the most part and I think it would be good to be consistent.

Of course, I prefer to use `pathlib.Path`, using `os` follows the previous call more.